### DISCUSSION TITLE
Member must have length less than or equal to 51200

### DISCUSSION THREAD

**Discussion by dannypaz** [serverless/serverless#10360] [General]
After upgrading from 2.59 to 2.69, I started receiving errors on one of my services 1 validation error detected: Value <MY SUPER RIDICULOUS LARGE CF TEMPLATE> at 'templateBody' failed to satisfy constraint: Member must have length less than or equal to 51200. 

Is there a work around to allow me to upload the CF template to s3 to get around this limitation? Can anyone point me to the code change that changed how templates were uploaded?

**Comment by pgrzesik** [comment]
Hello @dannypaz 👋 That's indeed a weird behavior. Were you able to inspect and compare the generated CF templates between the versions to see what is the difference? 

Also, could you please share the full command output for the message?

**Comment by lkzimmermann** [comment]
I had something similar happening to me, the reason was that I had specified a custom deployment bucket and took it out later, that's when I got this same message. Simply putting back in the original deployment bucket made it work. Might be something different but the error was the same.
